README.rst

```diff
@@ -99,21 +96,24 @@ Make the following changes to the ``staging.yml`` file:
 Start staging server
 ====================
 
-To start a Buildbot master on this machine run::
-
-   $ fab start:staging.yml
+Once the Docker image has built on the staging server, and the staging.yml file has been created, start the test Buildbot master from the local machine using::
 
-To update a slave on this machine, run::
+   $ fab restart:staging.yml
```

Comment:
Using `restart` is a bit of a hack to avoid pulling an image from dockerhub. Maybe explain that here.